source/administration/object-management/transition-objects-to-s3.rst

```diff
@@ -124,20 +124,20 @@ Procedure
 2) Configure the Remote Storage Tier
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Use the :mc-cmd:`mc admin tier add` command to add an Amazon S3 service as the
+Use the :mc-cmd:`mc ilm tier add` command to add an Amazon S3 service as the
 new remote storage tier:
 
 .. code-block:: shell
    :class: copyable
 
-   mc admin tier add s3 TARGET TIER_NAME \
+   mc ilm tier add s3 TARGET TIER_NAME \
```

Comment:
Could add the new limit upload/download flags.

Reply:
I added, since they are new flags. It would be good to get their use out there.